b2share/modules/schemas/alembic/35d7d8958395_create_schemas_tables.py

```diff
@@ -0,0 +1,99 @@
+"""Create schemas tables.
+
+Revision ID: 35d7d8958395
+Revises: 226892354af5
+Create Date: 2017-04-25 18:03:15.919582
+
+"""
+from alembic import op
+from datetime import datetime
+import sqlalchemy as sa
+import uuid
+
+from sqlalchemy.types import TIMESTAMP
+from sqlalchemy_utils.types import UUIDType
+from sqlalchemy.engine.reflection import Inspector
+
+
+# revision identifiers, used by Alembic.
+revision = '35d7d8958395'
+down_revision = '226892354af5'
+branch_labels = ()
+depends_on = 'fb99eeaec4ac'
```

Comment:
Could you add a comment mentioning that this comes from B2SHARE's communities module.